src/Caching/Test/ClassesTests.cs

```diff
@@ -152,7 +152,6 @@ class B(A):
         }
 
         [TestMethod, Priority(0)]
-        [Ignore("Todo: super() persistence support, Asserts in debug due to no model found for super")]
         public async Task ClassesWithSuper() {
```

Comment:
should I move these super tests to its own file?

Reply:
Might be a good idea, yeah.